deployments/get/objects.go

```diff
@@ -44,17 +44,17 @@ func GetClusterRole() *rbacv1.ClusterRole {
 			{
 				APIGroups: []string{""},
 				Resources: []string{"pods", "nodes", "namespaces", "configmaps"},
-				Verbs:     []string{"get", "patch", "list", "watch", "update"},
+				Verbs:     []string{"get", "list", "watch", "update"},
```

Comment:
we can remove `update` also

Reply:
we are updating namespaces [here](https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/core/kubeUpdate.go#L2433), so I'll create a different ruleset for namespaces.